src/rdf4cpp/rdf/datatypes/owl.hpp

```diff
@@ -0,0 +1,7 @@
+#ifndef RDF4CPP_OWL_HPP
+#define RDF4CPP_OWL_HPP
```

Comment:
add another header for rdf (for the sake of consistency)

Reply:
not sure if you meant that but found and fixed a include guard clash with namespaces/owl

Reply:
ah, it's there already. the rdf.hpp file